src/ResourceManager/TrafficManager/Commands.TrafficManager2/Commands.TrafficManager.csproj

```diff
@@ -111,6 +104,21 @@
       <SubType>Designer</SubType>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.TrafficManager, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@allencal you'll need to update the version of the SDK used in the [.NET Core project](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/TrafficManager/Commands.TrafficManager2/Commands.TrafficManager.Netcore.csproj#L35) for the Travis build to pass